packages/lesswrong/components/search/SearchBarResults.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import { registerComponent, Components } from 'meteor/vulcan:core';
+import PropTypes from 'prop-types';
+import {
+  Hits,
+  Configure,
+  Index } from 'react-instantsearch/dom';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  root: {
+    backgroundColor: theme.palette.background.default,
+    transition: "opacity .1s ease-in-out",
+    height: 2000,
+    width:"100%",
+    overflow: "hidden",
+    position: "absolute",
+    left: 0,
+    top: 64,
+    [theme.breakpoints.down('small')]: {
+      top:48,
+    }
+  },
+  searchResults: {
+    maxWidth: 1200,
+    margin: "auto",
+  },
+  searchList: {
+    padding:theme.spacing.unit*2
+  },
+  header: {
+    display:"flex",
+    justifyContent:"space-between",
+    alignItems: "center",
+    paddingTop: theme.spacing.unit*2,
+    paddingBottom: theme.spacing.unit*2,
+    paddingLeft: theme.spacing.unit,
+    paddingRight: theme.spacing.unit,
+    '& h1': {
+      margin:0
+    }
+  },
+})
+
+class SearchBarResults extends Component {
+
+  render() {
+    const { classes } = this.props
+
+    return <div className={classes.root}>
+      <Grid container className={classes.searchResults}>
```

Comment:
Do we want to start moving towards using the material-ui grid for layout in general? I noticed that sometimes we use it but sometimes we don't.